Source/WebGPU/WebGPU/HardwareCapabilities.mm

```diff
@@ -34,10 +34,10 @@
 
 namespace WebGPU {
 
-static constexpr uint32_t maxUInt32Limit()
+static constexpr uint32_t largeReasonableLimit()
 {
     // FIXME: https://bugs.webkit.org/show_bug.cgi?id=252873
-    return std::numeric_limits<uint32_t>::max() - 1;
+    return 100000;
```

Comment:
Does this require a comment stating it was pulled out of thin air?

Reply:
I kind of think the new function name indicates that via the first part of the name `largeReasonable`